### PR TITLE
Adds suppress ui flag to command line.

### PR DESCRIPTION
**Pull request by karthiknadig**
https://github.com/Microsoft/RTVS/issues/2702

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -67,10 +68,12 @@ namespace rhost {
             idle_timeout("rhost-idle-timeout", po::value<std::chrono::seconds::rep>(), (
                 "Shut down the host if it is idle for the specified duration in seconds. "
                 "If " + rdata.long_name() + " was specified, save workspace before exiting."
-                ).c_str());
+                ).c_str()),
+            suppress_ui("suppress-ui", new po::untyped_value(true),
```
Comment (int19h) [inline src/main.cpp]: I believe you can just omit the value here, and only have the name and the docstring.

File: src/main.cpp [hunk 2]
```diff
@@ -122,6 +125,12 @@ namespace rhost {
             args.idle_timeout = std::chrono::seconds(n);
         }
 
+        if (vm.count(suppress_ui.long_name())) {
```
Comment (int19h) [inline src/main.cpp]: `args.suppress_ui = vm.count(...) != 0`

File: src/main.cpp (hunk 1 above)
Comment (karthiknadig) [inline src/main.cpp]: Does't look like it supports that form of constructor:
```
option_description();
option_description(const char* name, const value_semantic* s);
option_description(const char* name, const value_semantic* s, const char* description);
```

**Comment by int19h** [comment]
👍

**Pull request merged**